deploy/resources/v0.8.0/addons/pipelinetemplates/s2i-java-11/s2i-java-11-build-deploy.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Pipeline
+metadata:
+  name: s2i-java-11-build-and-deploy-pipeline
+  namespace: openshift
+  labels:
+    pipeline.openshift.io/runtime: java
+spec:
+  params:
+    - name: IMAGE_NAME
+      type: string
+  resources:
+    - name: app-source
+      type: git
+    - name: app-image
+      type: image
+
+  tasks:
+    - name: build
+      taskRef:
+        name: s2i-java-11
+        kind: ClusterTask
+      resources:
+        inputs:
+          - name: source
+            resource: app-source
+        outputs:
+          - name: image
+            resource: app-image
+      params:
+        - name: TLSVERIFY
+          value: "false"
+
+    - name: deploy
+      taskRef:
+        name: openshift-client
+        kind: ClusterTask
+      params:
+        - name: ARGS
+          value:
```

Comment:
nit: `value: [ new-app, --docker-image, $(params.IMAGE_NAME) ] ` may be easier to read